### PR TITLE
[Review] fix(tools): remove deprecated NAMESPACE_MAP from cmake macros

### PR DESCRIPTION
**Pull request by GoetzGoerisch**
Follow-up to #6214
ping @NoelGraf

### CODE REVIEW

File: tools/cmake/macros_public.cmake [hunk 1]
```diff
@@ -612,7 +612,7 @@ function(ua_generate_nodeset_and_datatypes)
             NAME "types_${UA_GEN_NAME}"
             TARGET_PREFIX "${UA_GEN_TARGET_PREFIX}"
             TARGET_SUFFIX "types-${UA_GEN_NAME}"
-            NAMESPACE_MAP "${NAMESPACE_MAP_DEPENDS}"
+            # NAMESPACE_MAP "${NAMESPACE_MAP_DEPENDS}"
```
Comment (NoelGraf) [inline tools/cmake/macros_public.cmake:615]: Remove the argument instead of commenting it out.

File: tools/cmake/macros_public.cmake [hunk 2]
```diff
@@ -530,7 +530,7 @@ function(ua_generate_nodeset_and_datatypes)
 
     set(options INTERNAL AUTOLOAD)
     set(oneValueArgs NAME FILE_NS FILE_CSV FILE_BSD OUTPUT_DIR TARGET_PREFIX BLACKLIST)
-    set(multiValueArgs DEPENDS IMPORT_BSD NAMESPACE_MAP)
```
Comment (NoelGraf) [inline tools/cmake/macros_public.cmake:533]: The argument is being used elsewhere in the file. They should all be removed.

File: tools/cmake/macros_public.cmake (hunk 1 above)
Comment (GoetzGoerisch) [reply tools/cmake/macros_public.cmake]: Left over, removed.

File: tools/cmake/macros_public.cmake (hunk 2 above)
Comment (GoetzGoerisch) [reply tools/cmake/macros_public.cmake:533]: Thx, removed.

**Comment by jpfr** [comment]
Nice, thanks.

**Pull request merged**